src/xdocs/config_filters.xml

```diff
@@ -986,5 +986,277 @@ public class UserService {
           <p> <a href="config.html#TreeWalker">TreeWalker</a> </p>
         </subsection>
       </section>
+
+    <section name="SuppressWithPlainTextCommentFilter">
+          <subsection name="Description">
+              <p>Since Checkstyle 8.4</p>
```

Comment:
This is a new filter so all versions should say 8.6 .

Reply:
Done